packages/deb/bionic/kubelet/lib/systemd/system/kubelet.service

```diff
@@ -1,8 +1,9 @@
 [Unit]
 Description=kubelet: The Kubernetes Node Agent
 Documentation=https://kubernetes.io/docs/home/
-Wants=network-online.target
-After=network-online.target
+Wants=network-online.target time-sync.target
+After=network-online.target time-sync.target
+RequiresMountsFor=/var/lib/kubelet
```

Comment:
i don't think we release from the "bionic" specs. release eng team to confirm.

Reply:
Seems all per-distro are linked to bionic:

```
$ ls -la packages/deb/
total 60
drwxrwxr-x 3 hswong3i hswong3i  4096 Aug 25 12:58 .
drwxrwxr-x 4 hswong3i hswong3i  4096 Aug 25 12:58 ..
drwxrwxr-x 7 hswong3i hswong3i  4096 Aug 25 12:58 bionic
-rw-rw-r-- 1 hswong3i hswong3i   884 Aug 25 12:58 BUILD.bazel
-rw-rw-r-- 1 hswong3i hswong3i 14677 Aug 25 12:58 build.go
-rw-rw-r-- 1 hswong3i hswong3i  2618 Aug 25 12:58 build_test.go
-rw-rw-r-- 1 hswong3i hswong3i  1066 Aug 25 12:58 Dockerfile
-rw-rw-r-- 1 hswong3i hswong3i     5 Aug 25 12:58 .gitignore
-rw-rw-r-- 1 hswong3i hswong3i    97 Aug 25 12:58 go.mod
-rw-rw-r-- 1 hswong3i hswong3i   191 Aug 25 12:58 go.sum
-rwxrwxr-x 1 hswong3i hswong3i  1110 Aug 25 12:58 jenkins.sh
lrwxrwxrwx 1 hswong3i hswong3i     6 Aug 25 12:58 jessie -> bionic
-rw-rw-r-- 1 hswong3i hswong3i   221 Aug 25 12:58 OWNERS
lrwxrwxrwx 1 hswong3i hswong3i     6 Aug 25 12:58 sid -> bionic
lrwxrwxrwx 1 hswong3i hswong3i     6 Aug 25 12:58 stretch -> bionic
lrwxrwxrwx 1 hswong3i hswong3i     6 Aug 25 12:58 trusty -> bionic
lrwxrwxrwx 1 hswong3i hswong3i     6 Aug 25 12:58 xenial -> bionic
```